Sender/Sender_BMS_Data.c

```diff
@@ -0,0 +1,25 @@
+#include "Sender_BMS_Data.h"
+
+/********This Function will prints the battery parameters & displayed in console************************/
+void DisplayToConsole(char* buffer)
+{
+	printf("%s",buffer);

```

Comment:
How come receiver module will read this data

Reply:
Sender modules prints the data to console. Receiver has to read from console further process the data.